ambry-tools/src/main/java/com.github.ambry.tools/admin/PartitionManager.java

```diff
@@ -73,6 +79,12 @@ public static void main(String args[]) {
           parser.accepts("datacenterToAddReplicasTo", "The data center to which replicas need to be added to")
               .withOptionalArg().ofType(String.class);
 
+      String disableRackAwarenessFlag = "disableRackAwareness";
+      parser.accepts(disableRackAwarenessFlag, "Disable rack-aware partition allocation");
```

Comment:
why not accept this like the other arguments as a `ArgumentAcceptingOptionSpec`?